Upstream: submitted, https://savannah.gnu.org/patch/index.php?8183

From f64dc6b15533ea4a18f48829a3bfd02b7e9bce44 Mon Sep 17 00:00:00 2001
From: Benedikt Morbach <benedikt.morbach@googlemail.com>
Date: Thu, 19 Sep 2013 12:44:37 +0200
Subject: [PATCH] configure.ac: look for host-prefixed ar
Organization: Exherbo

This patch changes configure.ac to look for '${host}-ar' before looking for the unprefixed 'ar'.
This is useful when cross-compiling when all your binutils are prefixed.

I used AC_CHECK_TOOLS instead of AC_CHECK_TOOL, since autoconf docs say
that the former calls AC_SUBST.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 3303c53..6a9a3d5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -32,6 +32,7 @@ AC_EXEEXT
 AC_PROG_RANLIB
 AC_PROG_YACC
 gl_EARLY
+AC_CHECK_TOOLS([AR], [ar])
 
 AC_SYS_LARGEFILE
 AC_ISC_POSIX
-- 
1.8.4

